app/Models/Camp.php

```diff
@@ -393,7 +393,7 @@ public static function getCampSubscription($filter, $userid = null): array
                 }
             }
             if (count($child_camps) > 0) {
-                $camp_subs_child = CampSubscription::where('user_id', '=', $userid)->whereIn('camp_num', $child_camps)->where('topic_num', '=', $filter['topicNum'])->where('subscription_start', '<=', strtotime(date('Y-m-d H:i:s')))->where('subscription_end', '=', null)->orWhere('subscription_end', '>=', strtotime(date('Y-m-d H:i:s')))->get();
+                $camp_subs_child = CampSubscription::where('user_id', '=', $userid)->whereIn('camp_num', $child_camps)->where('topic_num', '=', $filter['topicNum'])->where('subscription_start', '<=', strtotime(date('Y-m-d H:i:s')))->where('subscription_end', '=', null)->orWhere('subscription_end', '>', strtotime(date('Y-m-d H:i:s')))->get();
                 $flag = ($camp_subs_child && sizeof($camp_subs_child) > 0);
                 if ($flag) {
                     $flag = 2;
```

Comment:
What is `2` is here? Can you please update it in the comments

Reply:
Flag 2 means that a child of a camp is subscribed. flag 1 means that camp is directly subscribed